### ISSUE TITLE
[3.0.0015] Cannot clear debug token details in Cached Publisher Summary dialog.

### ISSUE THREAD

**Issue by pvallabhaneni** [blackberry/VSPlugin#693]
Environment: Win 7 64bit and Win 8.1

Steps to Reproduce:
1. Go to Blackberry->Options
2. Go to Signing Page.
3. Click "Edit Publisher info..." button
4. Cached Publisher Summary dialog appears. Observe that the debug token details appear in the text boxes.
5. Click "Clear" button. Observe that the text boxes are cleared.
6. Click "OK" and open Cached Publisher Summary dialog again.
7. Observe that the debug token details are shown again.
8. Edit the text box (don't clear the text completely) and click OK button
9. Observe that changes persists when Cached Publisher Summary dialog is opened again.
10. Deleted all the debug tokens present in the system.
11. Go to Blackberry->Options
12. Go to Signing Page.
13. Click "Edit Publisher info..." button

Actual Result: Observe that the debug token details(Last shown details) are still shown in the text boxes even if the debug tokes are not present in the system.

Changes to the text are applying correctly when OK button is selected in Cached Publisher Summary dialog. for ex: Add a letter (1) to the end of author name and click ok. Change is applied and shown when dialog is opened again.

 But if the text boxes are cleared completely, then it is not applied when "OK" button is clicked on Cached Publisher Summary dialog.

**Comment by phofman** [comment]
Agree. The info was removed from registry, but remained in current session.

Check pull-request #694 for details.
